### PR TITLE
refact: changed some trivial JS codes to be more effective

### PR DESCRIPTION
**Pull request by ygnoh**
I changed some JS codes.
It could be trivial but I changed it to be more effective.

### CODE REVIEW

**Comment by ygnoh** [comment]
@billyvg Hello, could you check it out please?

File: src/sentry/static/sentry/vendor/simple-slider/simple-slider.js [hunk 1]
```diff
@@ -255,7 +255,7 @@ SimpleSlider.prototype.valueToRatio = function(value) {
     _ref = this.settings.allowedValues;
     for (idx = _i = 0, _len = _ref.length; _i < _len; idx = ++_i) {
       allowedVal = _ref[idx];
-      if (!(typeof closest !== "undefined" && closest !== null) || Math.abs(allowedVal - value) < Math.abs(closest - value)) {
+      if (closest == null || Math.abs(allowedVal - value) < Math.abs(closest - value)) {
```
Comment (billyvg) [inline src/sentry/static/sentry/vendor/simple-slider/simple-slider.js]: This is a vendor file, we shouldn't modify it.

File: tests/sentry/lang/javascript/example-project/launch.js [hunk 2]
```diff
@@ -4,8 +4,8 @@ const stacktraceParser = require('stacktrace-parser');
 
 function convertStack(frames) {
   return frames.map((item) => {
-    var match = item.file.match(/^.*\/(.*?)$/);
-    var fileName, absPath;
+    const match = item.file.match(/^.*\/(.*?)$/);
```
Comment (billyvg) [inline tests/sentry/lang/javascript/example-project/launch.js]: Honestly, I'm not familiar with where this file is actually used, I'd be more comfortable if you reverted changes to this file.

**Review by billyvg — CHANGES_REQUESTED** [review]

File: tests/sentry/lang/javascript/example-project/launch.js (hunk 2 above)
Comment (ygnoh) [reply tests/sentry/lang/javascript/example-project/launch.js]: In this file, `var`, `const` and `let` are mixed. So I changed `var`s to `const` or `let` to be better. Hmm.. 
but okay, I will revert this file.

File: src/sentry/static/sentry/vendor/simple-slider/simple-slider.js (hunk 1 above)
Comment (ygnoh) [reply src/sentry/static/sentry/vendor/simple-slider/simple-slider.js]: Okay :)

**Review by billyvg — APPROVED** [review]

**Pull request merged**